test/unit/forms/declaration/DepartureTransportSpec.scala

```diff
@@ -17,20 +17,21 @@
 package unit.forms.declaration
 
 import base.TestHelper
-import forms.declaration.DepartureTransport
+import forms.declaration.{DepartureTransport, TransportCodes}
 import forms.declaration.TransportCodes._
+import models.DeclarationType
+import models.declaration.Transport
 import unit.base.FormSpec
 
 class DepartureTransportSpec extends FormSpec {
```

Comment:
In general, I'd suggest testing the common logic once (using `onJourney` method for e.g.) and then for Clearance journey only test additional cases.

Reply:
The onJourney stuff is in the view specs, not the form.  I agree there are some duplicate tests here - I'll tidy them up.